readme.md

```diff
@@ -62,28 +78,62 @@ You can check the [default reporter](lib/reporter.js) to get the gist of how it
 
 You can fetch the scripts descriptions from a `package.json` in JavaScript.
 
-``` js
+<!--@example('example.js')-->
+```js
+'use strict';
 var getScriptsInfo = require('npm-scripts-info');
 var readPkgUp = require('read-pkg-up');
 
-readPkgUp
+readPkgUp()
   .then(function(result) {
      var scriptsInfo = getScriptsInfo(result.pkg);
-     console.log(result.scriptsInfo);
-     //> { test: 'Run the tests' }
+     console.log(scriptsInfo);
+     //> { md: 'Update the markdown files',
+     //    info: 'Display information about the scripts',
+     //    test: 'Run the tests' }
```

Comment:
Currently I cannot do it. That comment is inserted automatically by [this library](https://github.com/zkochan/codemo) that combines code with output. It currently inserts the output only on next line. All I can do it this version, if you like it more:

```js
console.log(JSON.stringify(scriptsInfo));
//> {"md":"Update the markdown files","info":"Display information about the scripts","test":"Run the tests"}
```